analysis/cross_partition_combiners.py

```diff
@@ -25,16 +25,16 @@ def _sum_metrics_to_data_dropped(
         sum_metrics: metrics.SumMetrics, partition_keep_probability: float,
         dp_metric: pipeline_dp.Metric) -> metrics.DataDropInfo:
     """Finds Data drop information from per-partition metrics."""
-    # TODO(dvadym): implement for Sum
-    assert dp_metric != pipeline_dp.Metrics.SUM, "Cross-partition metrics are not implemented for SUM"
 
     # This function attributed the data that is dropped, to different reasons
     # how they are dropped.
 
     # 1. linf/l0 contribution bounding
     # Contribution bounding errors are negative, negate to keep data dropped
     # to be positive.
-    linf_dropped = -sum_metrics.clipping_to_max_error  # not correct for SUM
+    linf_dropped = -sum_metrics.clipping_to_max_error
+    if dp_metric == pipeline_dp.Metrics.SUM:
+        linf_dropped += sum_metrics.clipping_to_min_error
```

Comment:
I don't negate it because it is positive in tests, is it correct or should I negate it here as well?

Also, I wonder whether for count and pid_count it is 0 and we can remove if condition and just always add it.

Reply:
1. Yes, clipping_to_max_error >= 0m and clipping_to_min_error <= 0, the idea that there is an invariant
`clipped_value = actual_value + clipping_to_max_error + clipping_to_min_error`
2. Yes, for count/privacy_id_count `clipping_to_min_error` is 0 (since not min cliping happens), let's drop `if`